backend/substrapp/tasks/tasks.py

```diff
@@ -181,6 +181,7 @@ def fetch_model(parent_tuple_type, authorized_types, input_model, directory):
         raise TasksError(f'{parent_tuple_type.capitalize()}: invalid input model: type={tuple_type}')
 
     model_dst_path = path.join(directory, f'model/{input_model["traintupleKey"]}')
+    raise_if_path_traversal([model_dst_path], path.join(directory, 'model/'))
```

Comment:
Since we're creating the model_dst_path just above from the directory and the (validated) traintupleKey, can't we just drop this check? I'm not seeing how it could fail and raise.

Reply:
The same would apply to all checks in this file, I'm clearly missing something, could you explain it to me?

Reply:
As we build the destination path base on the external value `input_model["traintupleKey"]` we (the backend) have no guarantee that `input_model["traintupleKey"]` is not path traversal, so should verify it :)